ACPs/226-dynamic-minimum-block-times/README.md

```diff
@@ -23,21 +23,21 @@ With the prospect of ACP-194 removing block execution from consensus and allowin
 
 ### Block Header Changes
 
-Upon activation of this ACP, the `blockGasCost` field in block headers will be required to be set to 0. This means that no validation of the cumulative priority fee amounts of transactions within the block exceeding the block gas cost is required. Additionally, two new fields will be added to EVM block headers: `timestampMilliseconds` and `minimumBlockDelayExcess`.
+Upon activation of this ACP, the `blockGasCost` field in block headers will be required to be set to 0. This means that no validation of the cumulative priority fee amounts of transactions within the block exceeding the block gas cost is required. Additionally, two new fields are added to EVM block headers: `timestampMilliseconds` and `minimumBlockDelayExcess`.
```

Comment:
```suggestion
Upon activation of this ACP, the `blockGasCost` field in block headers will be required to be set to 0. This means that the validation of the cumulative priority fee of transactions within the block exceeding the block gas cost is not required. Additionally, two new fields are added to EVM block headers: `timestampMilliseconds` and `minimumBlockDelayExcess`.
```

Reply:
could you open a separate PR for these improvement?, I don't think these are directly relevant to this PR.